pkg/cmd/cli/cmd/newapp.go

```diff
@@ -599,7 +599,7 @@ func retryBuildConfig(info *resource.Info, err error) runtime.Object {
 	return nil
 }
 
-func handleRunError(err error, baseName, commandName, commandPath string) error {
+func handleRunError(err error, baseName, commandName, commandPath string, config *newcmd.AppConfig) error {
```

Comment:
the only difference between this and handleBuildError is which function is invoked for transformation...seems like an opportunity to collapse some code (pass the transform function in as an argument)